### PR TITLE
I6990 makes guides addons titles linkable

### LINKED ISSUE #6990: Make addon titles on the new guide pages linkable

### PR DESCRIPTION
**Pull request by rebmullin**
fixes #6990

### CODE REVIEW

File: src/amo/components/GuidesAddonCard/styles.scss [hunk 1]
```diff
@@ -21,6 +21,11 @@
     font-size: $font-size-m;
     font-weight: 500;
     margin-bottom: 0;
+
+    a {
+      color: $black;
+      text-decoration: none;
```
Comment (bobsilverberg) [inline src/amo/components/GuidesAddonCard/styles.scss]: Do you think we should make the link change to blue on hover, like we do, for example, with search results?

Comment (rebmullin) [reply src/amo/components/GuidesAddonCard/styles.scss]: oh? let me check..

Comment (rebmullin) [reply src/amo/components/GuidesAddonCard/styles.scss]: yes, I like that 👍  thanks for the suggestion

File: src/amo/components/GuidesAddonCard/index.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
               <div className="GuidesAddonCard-content-header">
                 <div className="GuidesAddonCard-content-header-title">
                   <span className="GuidesAddonCard-content-header-authors">
-                    <AddonTitle addon={addon} as="span" />
+                    <AddonTitle addon={addon} as="span" linkToAddon />
```
Comment (willdurand) [inline src/amo/components/GuidesAddonCard/index.js]: ah, that's cool!

Comment (rebmullin) [reply src/amo/components/GuidesAddonCard/index.js]: yes all made very easy by the AddonTitle component:)

**Pull request merged**